continuous_integration/docker/hadoop/_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -xe
+
+cd /working
+pytest -v \
+    tests/test_yarn_backend.py \
```

Comment:
Silly question, but if we explicitly pick out yarn tests here, is there any point in the YARN_TEST (or whatever it's called) env variable? Should we set it here?

Reply:
That is a good point, I agree that we should instead rely on that instead, but it is a non-critical change so I may not drive making these tests skipped by default unless marked for execution - which could be what would be the cleanest strategy.

Reply:
I'm opting to not go for this refactoring, it can absolutely be done and I'll +1 for merging a PR about this, but for now I'll avoid doing that refactoring.